src/api/votes.ts

```
@@ -58,3 +58,45 @@ export const createVote = async (voteData: CreateVoteData) => {
     throw error
   }
 }
+
+export const fetchMineVotesCreated = async (
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)